spec/ride_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "./lib/ride.rb"
+
+RSpec.describe Ride do
+    before(:all) do 
+        @ride1 = Ride.new({name: "Walnut Creek Trail", distance: 10.7, loop: false, terrain: :hills})
+        @ride2 = Ride.new({name: "Town Lake", distance: 14.9, loop: true, terrain: :gravel})
+    end
+
+    describe "#initialize" do
+        it "can initialize" do
+            expect(@ride1).to be_an_instance_of (Ride)
+        end
+
+        it "can initialize wtih a name" do
+            expect(@ride1.name).to eq('Walnut Creek Trail')
+        end
+
+        it "can initialize with distance" do
+            expect(@ride1.distance).to eq(10.7)
+        end
+
+        it "can initialize with loop?" do
+            expect(@ride1.loop).to be false
```

Comment:
You had the question mark in the it statement, but it didn't make it's way to the actual method. REmember in Ruby, a method that returns a boolean should end in a `?`.